src/functions/meme/memes.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
We will need a way to access that list from Discord, maybe a command or something
Without autocomplete of command this will be insanly hard to remember

Reply:
I was thinking about thinking about linking to a page on the website, but that isn't ideal. Sending a huge list through discord messages isn't ideal either. Open to suggestions.

Reply:
We had the list to firestore then ?